Content.Shared/SS220/SwitchToggleItemSlots/SwitchToggleItemSlotsComponent.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Tools;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.SS220.SwitchToggleItemSlots;
+
+[RegisterComponent]
+[NetworkedComponent]
+public sealed partial class SwitchToggleItemSlotsComponent : Component
```

Comment:
Нейминг

```suggestion
public sealed partial class ToggleableItemSlotComponent : Component
```

Ну или `HiddenItemSlotComponent`